### PR TITLE
Move application data from ISIS3DATA to git repo for Voyager 1 and 2

### PR DESCRIPTION
**Pull request by scsides**
Move application data from ISIS3DATA to git repo for Voyager 1 and 2
## Description
Move application data from ISIS3DATA to git repo for Voyager 1 and 2
## Related Issue
#3727 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/appdata/translations/voyager.trn [hunk 1]
```diff
@@ -0,0 +1,204 @@
+Group = MissionPhaseName
```
Comment (kberryUSGS) [inline isis/appdata/translations/voyager.trn]: You could also put this at `ISIS3/isis/src/voyager/translations/` I think there was one other mission we did this with, and it seems slightly preferable to this location to me.

Comment (krlberry) [inline isis/appdata/translations/voyager.trn]: Why can't this go in `isis/voyager/apps/voy2isis` ?

File: isis/appdata/translations/voyager_na_bandbin.trn [hunk 2]
```diff
@@ -0,0 +1,75 @@
+#
```
Comment (krlberry) [inline isis/appdata/translations/voyager_na_bandbin.trn]: same as above

File: isis/appdata/translations/voyager_wa_bandbin.trn [hunk 3]
```diff
@@ -0,0 +1,109 @@
+
```
Comment (krlberry) [inline isis/appdata/translations/voyager_wa_bandbin.trn]: same as above

**Review by krlberry — CHANGES_REQUESTED** [review]
I think the translation files should get moved into `isis/voyager/apps/voy2isis/`? Let me know if I missed something.

File: isis/appdata/translations/voyager.trn (hunk 1 above)
Comment (scsides) [reply isis/appdata/translations/voyager.trn]: They can I put them in the wrong place.  Now moved

**Review by krlberry — APPROVED** [review]

**Pull request merged**